### PR TITLE
Feat darkmode

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the dark mode

In this PR, **111 additions 46 deletions** lines of code

### Please review this PR and give your feedback
---
#### [Live URL](https://react-marvels-quiz-git-feat-darkmode-hsnice16.vercel.app/)

image

### CODE REVIEW

**Review by BalajiHarinath — APPROVED** [review]
Everything looks good. Nothing to suggest.

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -14,8 +19,16 @@ export const Header = () => {
...
+      <button
+        onClick={handleThemeToggle}
+        className="btn-theme translate-y tooltip"
+      >
+        <i
+          className={classNames(theme === "dark" ? "fas" : "far", "fa-moon")}
+        ></i>
+        <span className="tooltip-left tooltip-text">
+          change to {theme === "dark" ? "white" : "dark"}
+        </span>
```
Comment (anik31) [inline src/components/header/Header.jsx:31]: appreciation - this tooltip is really nice, great job

File: src/pages/question/components/prev-next-link/PrevNextLink.jsx [hunk 2]
```diff
@@ -44,13 +44,13 @@ export const OtherPagesLink = ({ category, prev, next }) => {
   );
 };
 
-OtherPagesLink.propTypes = {
+PrevNextLink.propTypes = {
   category: PropTypes.string,
   prev: PropTypes.string,
   next: PropTypes.string,
 };
```
Comment (anik31) [inline src/pages/question/components/prev-next-link/PrevNextLink.jsx:51]: appreciation - good that you're checking data types of props

**Review by anik31 — APPROVED** [review]
Nice work @hsnice16 💯 The app looks good with dark mode & the toggle works perfectly. Code is also well structured & readable.

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**